src/seals/diagnostics/noisy_obs.py

```diff
@@ -31,9 +31,11 @@ def __init__(self, *, size: int = 5, noise_length: int = 20):
             state_space=spaces.MultiDiscrete([size, size]),
             action_space=spaces.Discrete(5),
             observation_space=spaces.Box(
-                low=np.concatenate(([0, 0], np.full(self._noise_length, -np.inf))),
+                low=np.concatenate(
```

Comment:
Why the change? Type checking passes fine locally for me with both `mypy` and `pytype` without these ignores. Presumably also passes on CI given checks passed on master.

Reply:
```
src/seals/diagnostics/noisy_obs.py:34: error: Argument 1 to "concatenate" has incompatible type "Tuple[List[int], ndarray[Any, dtype[Any]]]"; expected "Union[_SupportsArray[dtype[<nothing>]], _NestedSequence[_SupportsArray[dtype[<nothing>]]]]"
src/seals/diagnostics/noisy_obs.py:36: error: Argument 1 to "concatenate" has incompatible type "Tuple[List[int], ndarray[Any, dtype[Any]]]"; expected "Union[_SupportsArray[dtype[<nothing>]], _NestedSequence[_SupportsArray[dtype[<nothing>]]]]"
```

Reply:
But then I get a `line too long`...

Reply:
Huh, interesting, is this is a version difference between your machine and CI?